app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return
+    operation, source_file, new_file = command.split()
+    if operation != "mv":
+        return
```

Comment:
also try to refactor function to make no returns, for example, here you can check `if opertaion == "mv"`